app/serializers/application.js

```diff
@@ -5,4 +5,7 @@ export default DS.JSONAPISerializer.extend({
   keyForAttribute(attr) {
     return Ember.String.underscore(attr);
   },
+  keyForRelationship: function(key) {
+    return Ember.String.underscore(key);
+  },
```

Comment:
Switching it on the API side would be problematic, due to the way we write our specs and code in general. `:some_symbol` or `object.some_property` work fine in Ruby. `:some-symbol` and `object.some-property` not so much.